javascript/hash-table/hashTable.test.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const HashTable = require('./hashTable');
+
+describe('Hash Table', () => {
+  it('Setting a key/value to your hashtable results in the value being in the data structure', () => {
+    const myHashTable = new HashTable();
+    myHashTable.set('name', 'Emma');
+    myHashTable.set('age', 33);
+
+    expect(myHashTable.get('name')).toBe('Emma');
+    expect(myHashTable.get('age')).toBe(33);
+  });
+
+  it('Retrieving based on a key returns the value stored', () => {
+    const myHashTable = new HashTable();
+
+    myHashTable.set('name', 'Emma');
+    myHashTable.set('age', 33);
+
+    expect(myHashTable.get('name')).toBe('Emma');
+    expect(myHashTable.get('age')).toBe(33);
+  });
+
+  it('Successfully returns null for a key that does not exist in the hashtable', () => {
+    const myHashTable = new HashTable();
+
+    myHashTable.set('name', 'Emma');
+    myHashTable.set('age', 33);
+
+    expect(myHashTable.get('city')).toBeNull();
+  });
+
+  it('Successfully returns a list of all unique keys that exist in the hashtable', () => {
+    const myHashTable = new HashTable();
+
+    myHashTable.set('name', 'Emma');
+    myHashTable.set('age', 33);
+    myHashTable.set('dogs', 2);
+    myHashTable.set('city', 'Olympia');
+
+    const keys = myHashTable.keys();
+
+    expect(keys).toEqual(expect.arrayContainting(['name', 'age', 'dogs', 'city']));
```

Comment:
If you make the changes above or in the hash-table.js, then change line 42 to: ` const keys = myHashTable.key();`. Also, in line 44, you have an extra 't' arrayContaining